mp/src/game/server/logic_random_outputs.h

```diff
@@ -0,0 +1,51 @@
+//========= Copyright � 1996-2005, Valve Corporation, All rights reserved. ============//
+//
+// Purpose: 
+//
+// $NoKeywords: $
+//
+//=============================================================================//
+#ifndef LOGICRANDOMOUTPUTS_H
+#define LOGICRANDOMOUTPUTS_H
+
+#include "cbase.h"
+#include "entityinput.h"
+#include "entityoutput.h"
+#include "eventqueue.h"
```

Comment:
You do not want to include "cbase.h" in a header. Clean up these imports to only inside what you truly need, and forward declare those that you can.

The less includes in a header the better, as it leads to less accidental dependencies (other headers including this header when they needed what this header includes) which also speeds up compiling.